api/views.py

```diff
@@ -45,6 +47,15 @@ class ExportInstructorLocationsView(ListAPIView):
     serializer_class = ExportInstructorLocationsSerializer
 
 
+class ExportMembersView(ListAPIView):
+    """Show everyone who qualifies as an SCF member."""
+    permission_classes = (IsAuthenticatedOrReadOnly, )
+    paginator = None  # disable pagination
+
+    queryset = get_members()
```

Comment:
This line gets evaluated on every request because of this line: https://github.com/swcarpentry/amy/pull/577/files#diff-ef5bc9252746919e203b86c32d3d437eR14